Code/autopkglib/URLDownloader.py

```diff
@@ -163,136 +205,10 @@ def main(self):
         # with the same permissions and the file is inaccessible by (for example)
         # the webserver.
         os.chmod(pathname_temporary, 0o644)
+        return pathname_temporary
 
-        # construct curl command.
```

Comment:
Most of this code was moved to `prepare_curl_cmd()` and to `URLGetter` superclass.